app/pages/sections/routeDetails/RoutingSection.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.sections.routeDetails
+
+import pages.sections.Section
+import play.api.libs.json.{JsObject, JsPath}
+
+case object RoutingSection extends Section[JsObject] {
+
+  override def path: JsPath = RouteDetailsSection.path \ toString
```

Comment:
Should the path now also be RoutingSection or has it been intentionally left this way?

Reply:
@kevincoatsworth The plan will be to have the user answers data structured something like:
```
{
    "routeDetails": {
        "routing":{},
        "transit": {},
        "exit": {},
        "locationOfGoods": {},
        "loading": {},
        "unloading": {}
    }
}
```